java-squid/src/main/java/org/sonar/java/model/expression/TypeArgumentListTreeImpl.java

```diff
@@ -67,7 +67,8 @@ public void accept(TreeVisitor visitor) {
 
   @Override
   public Iterator<Tree> childrenIterator() {
-    return ImmutableList.<Tree>builder().addAll(this).build().iterator();
+    // FIXME SONARJAVA-547 separators between arguments are currently ignored
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Take the required action to fix the issue indicated by this comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1134)